.github/actions/turbo-diffs/action.yml

```diff
@@ -7,9 +7,6 @@ outputs:
 runs:
   using: composite
   steps:
-    - uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871 # v4.2.1
-      with:
-        fetch-depth: 0
```

Comment:
Please revert and create an issue

Reply:
https://github.com/iotaledger/iota/issues/5017